### ISSUE TITLE
Issue in setting up the bot

### ISSUE THREAD

**Issue by lilithxxx** [JasonEtco/ci-reporter#60]
I tried installing the bot on my repo but its not posting any comment on a PR with failing builds. I am using CircleCI for the tests. I also created a config file but it ain't helping either. Am I missing any steps?
Thanks

**Comment by lilithxxx** [comment]
@JasonEtco Ping? Can anybody help me please? Thanks

**Comment by JasonEtco** [comment]
It's only been two days, you don't need to ping me 😅 Can you please share some more details? What does your config file look like? Is it a private repository?

**Comment by lilithxxx** [comment]
I am just trying out the bot in my local repo so that I can use it in my org. The ci-reporter.yaml file is:
```
# Set to false to create a new comment instead of updating the app's first one
updateComment: true

# Use a custom string, or set to false to disable
before: false

# Use a custom string, or set to false to disable
after: false
```
And the circleci file is very simple and it's used to just test a single file :
```
# Python CircleCI 2.0 configuration file
#
# Check https://circleci.com/docs/2.0/language-python/ for more details
#
version: 2.1
jobs:
  build:
    docker:
      # specify the version you desire here
      # use `-browsers` prefix for selenium tests, e.g. `3.6.1-browsers`
      - image: circleci/python:2.7.14

      # Specify service dependencies here if necessary
      # CircleCI maintains a library of pre-built images
      # documented at https://circleci.com/docs/2.0/circleci-images/
      # - image: circleci/postgres:9.4

    working_directory: /home/circleci/hello-world

    steps:
      - checkout
 
      # run tests!
      # this example uses Django's built-in test-runner
      # other common Python testing frameworks include pytest and nose
      # https://pytest.org
      # https://nose.readthedocs.io
      - run:
          name: run file
          command: |
            python ex.py
      - run:
          name: run tests
          command: |
            sudo pip install coverage
            coverage run ex.py
           
      - run:
          name: Generate backend coverage report
          command: |
            sudo pip install codecov
            codecov
          when: on_success

      - store_artifacts:
          path: test-reports
          destination: test-reports
```
And no its not a private repo. Thanks

**Comment by JasonEtco** [comment]
It looks like this issue: https://github.com/JasonEtco/ci-reporter/issues/36

Where because your job is named `build`, it's not being picked up by CI Reporter.

**Comment by JasonEtco** [comment]
Closing as a duplicate of #36
